### PR TITLE
YLP-974 Add a filter on export service to export some types of data

### PR DESCRIPTION
**Pull request by CedricPerotto**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,4725 +1,8 @@
 {
   "name": "export",
   "version": "1.6.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```
Comment (ydrisR) [inline package-lock.json]: package lock version should be 2

**Review by BigFab — APPROVED** [review]
LGTM
I was able to test it as a container form "platforms_utils/ylp_scripts/export_users"
Works like a charm

File: data-tools.js [hunk 2]
```diff
@@ -282,8 +282,13 @@ export default class TidepoolDataTools {
     return JSONStream.parse('*');
   }
 
-  static tidepoolProcessor(processorConfig = {}) {
+  static tidepoolProcessor(processorConfig = {}, filteredType = false) {
```
Comment (ydrisR) [inline data-tools.js]: why not use an empty array instead of "false" by default? filteredType is suposed to be used as an array

**Review by ydrisR — CHANGES_REQUESTED** [review]
LGTM but a couple of comments

**Review by ydrisR — APPROVED** [review]
LGTM

**Pull request merged**